parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -5741,3 +5740,124 @@ SqlSelectInto SqlSelectInto() :
             fromClause, whereClause);
     }
 }
+
+SqlCall FirstLastValue() :
+{
+    final SqlNode value;
+    final SqlNode over;
+    final List<SqlNode> args = new ArrayList<SqlNode>();
+    SqlNullTreatmentModifier nullTreatmentModifier = null;
+    final SqlAggFunction function;
```

Comment:
Why use SqlAggFunction rather than SqlFirstLastValueAggFunction?

Reply:
I used SqlAggFunction since that is the type which is returned by the FIRST_VALUE and LAST_VALUE calls: https://github.com/googleinterns/calcite/blob/67b7bb66aabf05a58171096356dc71476bf76381/core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java#L977

Reply:
SGTM